src/makeCalculator.js

```diff
@@ -36,8 +36,49 @@
  *
  * @return {object}
  */
+
+const ops = {
+  result: 0,
+  // 'add': makeAdd,
+  // 'subtract': makeSubst,
+  // 'multiply': makeMult,
+  // 'divide': makeDiv,
+  // 'reset': makeReset,
+  // 'operate': makeOperate,
+
+};
+
+ops.operate = function makeOperate(operation, value) {
```

Comment:
better to move methods to the object and use descriptive variable name for it, smth like `calc`

Reply:
done